### ISSUE TITLE
seek bar on spotify -  incorrectly time 

### ISSUE THREAD

**Issue by piotrek2555** [michaelherger/Spotty-Plugin#102]
[img]https://i.ibb.co/1GMNjY9/Screenshot-2023-11-03-09-57-32-136-com-spotify-music.jpg [/img]

seek bar on spotify show incorrectly time, when i paused song and then click play, time seek bar show incorrectly time, but when switching on next song, time bar  is ok. LMS 8.3.1 - 1676361197  on raspberry pi 4b

**Comment by michaelherger** [comment]
Yes, that unfortunately is one of the many issues I'm seeing with the Connect feature integration. I'd have to come up with a totally new architecture to overcome this.

**Comment by dipgithub** [comment]
I have a similar problem with Spotify Connect. After a song has finished playing very often the time counter of the seek bar continues to increase indefinitely and the next song in the queue is not played. Unfortunately this makes Spotify Connect with Spotify almost unusuable for me.

**Comment by pwcutajar** [comment]
@michaelherger  is this integration dead then?

It's seems to not very well at all lately with LMS on my raspberry pis 2w and Spotify connect. It does however work flawless from squeezer app on android.

**Comment by michaelherger** [comment]
AFAIK it's as broken/working as it's been for years. I'm not aware of a situation which made it worse.

**Comment by pwcutajar** [comment]
I would have said that about a month ago (your disclaimer aside the spotty LMS integration was just about perfect!!!. But alas the last few weeks I've seen that seek doesn't work and sometimes (every few tracks ) the music just stops. Skipping tracks can help but not for long. I've also found that even though the device is in the connect list music doesn't play until the device is rebooted.
Again from inside squeezer this isn't an issue. 

I would admit there are more than one option in the integrations settings that Id be happy to take advice on if it will gets things working. I have multiple end points and 3 separate installs so taking the time would be worth it to me

**Comment by michaelherger** [comment]
> I would admit there are more than one option in the integrations settings that Id be happy to take advice on if it will gets things working. I have multiple end points and 3 separate installs so taking the time would be worth it to me

Please note that Connect mode will only ever work on a single device. You can't stream to multiple devices at the same time. Starting playback on one device would stop playback on any other.

**Comment by pwcutajar** [comment]
Sure noted but the whole Spotify connect function seems to be broken/ unstable

**Comment by dipgithub** [comment]
I would also appreciate of very much if the Spotify connect feature could be improved/repaired. As mentioned I have very often the problem that playback stops after a song while the time counter of this song increases indefinetely.
